_pages/algorithmia-enterprise/insights.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: article
+title:  "Insights"
+excerpt: "Sending algorithm metrics to a customer-operated metrics collection system"
+categories: algorithmia-enterprise
+show_related: false
+image:
+  teaser: /icons/algo.svg
+---
+
+Insights is an algorithm metrics pipeline providing you access to your algorithm metrics payload for each algorithm session so you can measure, monitor, and manage your algorithms in production.
+
+With Insights you'll be able to opt-in to exporting your algorithm's operational metrics to external monitoring and alerting tools. This means you’ll be able to monitor your algorithm's execution time and memory consumption as well as capture any model inference metrics you expose from your algorithm's output such as predictions or accuracy. Any metrics you opt to collect will be captured and exported in a payload and, with help from a Platform Administrator, can be connected to your external systems for monitoring and alerting.
+
+Insights currently supports [Python 3.x](../../clients/python) and [R](../../clients/r) algorithms and [Apache Kafka](https://kafka.apache.org/) as a destination for metrics.
+
+### Types of Insights metrics
+
+Insights metrics fall into two categories:
+* Operational metrics
+* Inference-related metrics
+
+**Operational metrics** include (but are not limited to) execution time, memory consumption, request id, algorithm owner, algorithm name, algorithm version, timestamp, etc. When Insights are turned on for a particular algorithm version, these are automatically collected and reported without any additional action necessary on the part of the algorithm author / manager.
+
+**Inference-related metrics** are defined by the algorithm author in algorithm code. Each metric is defined by a title, like `cute_cats_detected` and a numeric value, like `24`. Multiple inference-related metrics can be reported at one time, and the set of metrics reported does not need to be the same from one algorithm execution to another.
+
+## One-time administrator setup
+
+The first step in allowing Insights to be collected and reported for algorithms is to have the Algorithmia Platform Administrator connect your cluster to a Kafka broker that your organization operates to receive metrics created by the Algorithmia platform.
+
+Navigate to the _Insights_ administration page, which can be found under the "System Actions" section.
```

Comment:
I am calling the page 'Algorithm Insights' in the UI to be clear it's not platform insights.

Reply:
OK. Is platform insights documented in the dev center so I can read and educate myself about it?

Reply:
No, not specifically just want the tab to be clear what you're clicking on

Reply:
Ok, I'll change this wording here. Thanks!